scalopus_tracing/src/native/native_trace_source.cpp

```diff
@@ -149,6 +167,23 @@ std::vector<json> NativeTraceSource::finishInterval()
     return lhs.at("ts").get<double>() < rhs.at("ts").get<double>();
   });
 
+  // Need a reverse iteration here, to populate all counters with all series seen in the entire interval.
+  ProcessCounter counte_all_series;
+  for (auto it = res.rbegin(); it < res.rend(); it++)
```

Comment:
Same comment as for lttng implementation

Reply:
Same reasoning, it's a reverse iterator.